.github/workflows/prerelease.yml

```
@@ -133,6 +133,11 @@ jobs:
           slack-bot-user-oauth-access-token: ${{ secrets.COREINT_SLACK_TOKEN }}
           slack-channel: ${{ secrets.COREINT_SLACK_CHANNEL }}
           slack-text: "❌ `${{ env.REPO_FULL_NAME }}`: prerelease pipeline failed."
+      - name: Test package installability
+        uses: newrelic/integrations-pkg-test-action/linux@v1.0.0
```

I think it's better if we point to `v1`, so we can make backwards-compatible changed on everything at the same time
```suggestion
        uses: newrelic/integrations-pkg-test-action/linux@v1
```